package.json

```diff
@@ -59,18 +59,14 @@
     "email-prop-type": "1.1.7",
     "font-awesome": "4.7.0",
     "form-urlencoded": "3.0.0",
-    "glob": "7.1.3",
     "history": "4.7.2",
-    "i18n-iso-countries": "3.7.8",
-    "iso-countries-languages": "0.2.1",
```

Comment:
This is meant to just go away?

Reply:
Yes.  They are unused - they got moved into frontend-i18n